components/playlist/browser/playlist_service.cc

```diff
@@ -51,6 +49,32 @@ std::vector<base::FilePath> GetOrphanedPaths(
   return orphaned_paths;
 }
 
+std::string ToString(mojom::PlaylistEvent type) {
+  switch (type) {
+    case mojom::PlaylistEvent::kItemAdded:
+      return "item: added";
+    case mojom::PlaylistEvent::kItemDeleted:
+      return "item: deleted";
+    case mojom::PlaylistEvent::kItemAborted:
+      return "item: aborted";
+    case mojom::PlaylistEvent::kItemThumbnailReady:
+      return "item: thumbnail_ready";
+    case mojom::PlaylistEvent::kItemThumbnailFailed:
+      return "item: thumbnail_failed";
+    case mojom::PlaylistEvent::kItemCached:
+      return "item: cached";
+    case mojom::PlaylistEvent::kListCreated:
+      return "list: created";
+    case mojom::PlaylistEvent::kAllDeleted:
+      return "item: all deleted";
+    case mojom::PlaylistEvent::kNone:
+      [[fallthrough]];
+    default:
+      NOTREACHED() << "Unknown type: " << static_cast<int>(type);
```

Comment:
nit: maybe we could use `NOTREACHED_NORETURN()` here instead of unused return string?